docs/product/issues/issue-details/sentry-seer/index.mdx

```diff
@@ -0,0 +1,70 @@
+---
+title: Generative AI in Sentry
+description: "Learn about Seer, Sentry's AI agent, and managing generative AI features."
+sidebar_order: 50
+---
+
+Seer is Sentry's AI agent that powers a set of features built on generative AI and ML. Once you've set up Seer, you'll be able to use the following features:
+
+- [**Autofix**](#autofix): A collaborative workflow to find the root cause and solution of issues
+- [**Issue Summary**](#issue-summary): A quick summary of issue context to help you see what's going on
+
+<Alert>
+
+Autofix and Issue Summary are currently in Beta, free of charge, for paid Sentry customers. Feedback is highly welcomed and can be submitted in-product or via email to [autofix@sentry.io](mailto:autofix@sentry.io).
+
+</Alert>
+
+### Autofix
+
+**Autofix** uses Sentry's context (issue details, tracing data, and profiles), your codebase (integrated through GitHub), and its interactions with you to identify and reason through problems in your code. Autofix can identify the root cause and suggest a code change for fixing the issue and/or add unit tests for it.
+
+<Alert>
+
+Autofix replaces Suggested Fix (an earlier feature that has been sunset).
+
+</Alert>
+
+When you start an Autofix run, you'll see the following screen, with an option to share any additional context:
+
+![Autofix Start Screen](./img/autofix-start-screen.png)
+
+Autofix will then analyze the issue and relevant code to help identify the root cause and a solution. You'll see a live stream of Autofix's thought process in real time as it works through the problem. Autofix will share key insights as they come up and you'll have the option to expand each. You'll see the event data and code that support Autofix conclusions, building a clear chain of reasoning back to the root cause.
+
+While Autofix is reasoning through your problem, you can step in at any time to add context or feedback; collaboration tends to lead to the best results. If Autofix encounters gaps in its understanding, it will ask for your input.
+
+The output will be a clearly defined root cause of the Sentry issue, and a detailed solution for how to resolve it:
+
+![Autofix Insights](./img/autofix-qna.png)
+
+You are free to edit or adjust this solution based on your own requirements and preferences. Once you are ready to proceed, you can select "Code It Up" to generate code resolving the bug. You have three options:
+
+- Write a fix
+- Write a unit test reproducing the issue
```

Comment:
this sounds like a remnant for when we had a dropdown. Now we let the user add/remove any steps they want. Maybe could be worded better?

Reply:
i'll comment on it for rachel, i'm not sure